tests/ssp/enc/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+//# Copyright (c) Open Enclave SDK contributors.
+//# Licensed under the MIT License.
+
+oeedl_file(../ssp.edl enclave gen)
```

Comment:
Augh we really gotta remove `oeedl_file` so people stop using it...

Reply:
Yes, our CMake SDK documentation has it right here: https://github.com/openenclave/openenclave/blob/master/cmake/sdk_cmake_targets_readme.md#running-oeedger8r

Reply:
And I just filed #2325.

Reply:
Updated, oeedl_file is removed.